Resources/doc/operations.md

```diff
@@ -20,30 +20,9 @@ By default, the following operations are automatically enabled:
 
 ## Disabling operations
 
-If you want to disable some operations (e.g. the `DELETE` operation), you must register manually applicable operations using
-the operation factory class, `Dunglas\ApiBundle\Resource::addCollectionOperation()` and `Dunglas\ApiBundle\Resource::addCollectionOperation()`
-methods.
+You can specify the list of operation to enable by setting two properties `collectionOperations` and `itemOperations`.
```

Comment:
Please give a context of why you may want to do that, just an example, to make it clearer for beginners.